user.go

```diff
@@ -96,6 +96,15 @@ type user struct {
 
 	networks        []*network
 	downstreamConns []*downstreamConn
+
+	pendingListsLock sync.Mutex
+	pendingLists     []pendingList
```

Comment:
```go
// LIST commands in progress
```

Reply:
Maybe we could even rename this to `pendingLISTs` to make it clear it's about LIST, and not just a list?